library/ssl_misc.h

```diff
@@ -1415,6 +1441,43 @@ static inline int mbedtls_ssl_conf_tls13_some_psk_enabled( mbedtls_ssl_context *
                    MBEDTLS_SSL_TLS13_KEY_EXCHANGE_MODE_PSK_ALL ) );
 }
 
+static inline unsigned mbedtls_ssl_tls1_3_check_kex_modes( mbedtls_ssl_context *ssl,
```

Comment:
```suggestion
/*
 * Check if at least one of the defined key exchange modes is supported.
 *
 * \param[in] ssl  SSL context
 * \param key_modes_mask  Mask of the key exchange modes to check
 *
 * \return 0 if at least one of the defined key exchange modes is supported, <>0 otherwise.
 */
static inline unsigned mbedtls_ssl_tls1_3_check_kex_modes( mbedtls_ssl_context *ssl,
```